baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -388,7 +400,8 @@ def init(self, req):
 
         # install noVNC
         if not os.path.exists(self.NOVNC_INSTALL_PATH):
-            ret = bash_r("tar -xf %s -C %s" % (os.path.join(self.BAREMETAL_LIB_PATH, "noVNC.tar.gz"), self.BAREMETAL_LIB_PATH))
+            ret = bash_r(
```

Ruff (0.8.2)

`bash_r` may be undefined, or defined from star imports

(F405)